<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmulatorSettings</class>
 <widget class="QDialog" name="EmulatorSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Emulator Configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="emulator_tab">
      <attribute name="title">
       <string>Emulator</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QComboBox" name="comboBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="editable">
            <bool>false</bool>
           </property>
           <item>
            <property name="text">
             <string>Mupen64Plus</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="config_emu_btn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Configure</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="controller_tab">
      <attribute name="title">
       <string>Controller</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="Frontend::EmulatorControllerConfiguration" name="controller_widget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="video_tab">
      <attribute name="title">
       <string>Video</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="Frontend::EmulatorVideoConfiguration" name="video_widget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="audio_tab">
      <attribute name="title">
       <string>Audio</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="Frontend::EmulatorAudioConfiguration" name="audio_widget" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Frontend::EmulatorAudioConfiguration</class>
   <extends>QWidget</extends>
   <header>emulator_audio_configuration.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Frontend::EmulatorVideoConfiguration</class>
   <extends>QWidget</extends>
   <header>emulator_video_configuration.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Frontend::EmulatorControllerConfiguration</class>
   <extends>QWidget</extends>
   <header>emulator_controller_configuration.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
